docs/setup-and-installing.md

```diff
@@ -2,152 +2,79 @@
+    - If you are using Visual Studio, within the Visual Studio Installer, on the **Workloads** tab:
+        - Select **Game development with Unity**
+        - Select **.NET Core cross-platform development**
+        - In the **Summary** on the right:
+            - Deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.8)
+            - Make sure **Visual Studio Tools for Unity** is selected
+    - If you are using Rider:
+        - Install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience
+1. [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (Verified with version `2.1.3xx` or `2.1.4xx`)
+1. SpatialOS, using the the [SpatialOS installer](https://console.improbable.io/installer/download/stable/latest/win). This installs:
+    - the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/spatial-cli-introduction)
+    - the SpatialOS [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher)
+    - the 32-bit and 64-bit Visual C++ Redistributables
+1. Reboot your machine to finish the setup.
+
+> **Warning**: Older versions of Visual Studio 2017 have been known to cause some issues with Unity 2018.2.8 - the issues are projects loading and unloading frequently, and Intellisense breaking. If you do experience these issues, try updating to a newer version of Visual Studio 2017.
+
+#### Mac
+
+1. [Unity 2018.2.8](https://unity3d.com/get-unity/download/archive)
+    - Make sure to select the following components during installation:
+        - **Linux Build Support**
+        - **Windows Build Support**
+1. [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Verified with version `7.6.5`) or [Rider](https://www.jetbrains.com/rider/) (Verified with version `2018.2.1`)
+    - If you are using Visual Studio, within the Visual Studio Installer, select **.NET Core + ASP .NET Core**
+1. [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (Verified with version `2.1.3xx` and `2.1.4xx`)
+1. SpatialOS, using the the [SpatialOS installer](https://console.improbable.io/installer/download/stable/latest/mac). This installs:
+    - The [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/spatial-cli-introduction)
+    - The SpatialOS [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher)
+    - The 32-bit and 64-bit Visual C++ Redistributables
+1. Reboot your machine to finish the setup.
+
+### Clone the SpatialOS GDK for Unity repository
+
+|     |     |
+| --- | --- |
+| SSH | `git clone git@github.com:spatialos/UnityGDK.git` |
+| HTTPS | `git clone https://github.com/spatialos/UnityGDK.git` |
+
+The GDK repository is a SpatialOS project called `UnityGDK`. It contains:
+
+- a Unity project at `UnityGDK/workers/unity`, which you need to open to use the GDK
+- SpatialOS features, such as the schema and snapshot files
```

Comment:
Note to self - when we have made the GDK glossary (WIP) - add links to these terms from here - not helpful to link away from GDK docs at this point I think.